Tasks/Maven/Strings/resources.resjson/de-de/resources.resjson

```diff
@@ -20,5 +20,11 @@
   "loc.input.label.jdkUserInputPath": "JDK Path",
   "loc.input.help.jdkUserInputPath": "Sets JAVA_HOME to the given path.",
   "loc.input.label.jdkArchitecture": "JDK-Architektur",
-  "loc.input.help.jdkArchitecture": "Geben Sie optional die JDK-Architektur an (x86, x64)."
+  "loc.input.help.jdkArchitecture": "Geben Sie optional die JDK-Architektur an (x86, x64).",
```

Comment:
typically we don't check non-English strings in.  They generated English strings file get's handed off to loc and then they check back in.